cfme/tests/v2v/test_v2v_ansible.py

```diff
@@ -170,4 +167,10 @@ def _cleanup():
     view.flash.assert_no_error()
 
     migrated_vm = get_migrated_vm(src_vm_obj, provider)
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
```

Comment:
Hmmm...
I do not like that the mapping is left without any finalizer until here. I was trying to figure out why you are registering the finalizer after line 169, but did not succeed. Can you please explain?

Reply:
@jaryn :  I have added another line in finalizer method that needs `migrated_vm` which is at line 169 so moved the finalizer down